release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ocp-4-16-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+Do not add or edit release notes here. Edit release notes directly in the branch
+that they are relevant for.
```

Comment:
Is there a reason that this is hard wrapped?

Reply:
:heavy_check_mark: